scripts/commands/addtreasure.lua

```diff
@@ -9,28 +9,43 @@ cmdprops =
     parameters = "isi"
 };
 
+function error(player, msg)
```

Comment:
Overall I like the idea of just standardizing the usage error handling in all the commands like this.